### ISSUE TITLE
complaints about format of operator "*" in @specs are in conflict with mix format

### ISSUE THREAD

**Issue by pdobacz** [rrrene/credo#583]
### Environment

* Credo version (`mix credo -v`): 0.10.0
* Erlang/Elixir version (`elixir -v`):
```
Erlang/OTP 20 [erts-9.3.3.3]
Elixir 1.7.2 (compiled with Erlang/OTP 20)
```
* Operating system: Ubuntu 16.04

### What were you trying to do?

I'm trying to `mix format` and `mix credo` the following typespec:

```
@spec f(<<_::16, _::_*8>>) :: binary
```
This is the way `mix format` likes it

### Expected outcome

Credo should adhere to `mix format` and prefer the no-space version, but not sure. In any case they should play along.

### Actual outcome

credo complains:
```
┃ [C] ↗ There are spaces around operators most of the time, but not here.
```
and prefers:
```
@spec f(<<_::16, _::_ * 8>>) :: binary
```

**Comment by rrrene** [comment]
Thx for reporting! :+1:

**Comment by rrrene** [comment]
Thanks for reporting this 😀  It is now fixed on `master`.

You can try this by setting the Credo dep to

```elixir
{:credo, github: "rrrene/credo"}
```

Please report back if your issue is solved! 👍

**Comment by pdobacz** [comment]
@rrrene Thank you, the issue appears to be fixed in `master`!

However, I might have bumped into some version of https://github.com/rrrene/credo/issues/575 with `master`. I am not able to make `credo` not complain about the aliases. I'll post mine there
